MAUI/Calendar/getting-started.md

```diff
@@ -156,4 +156,34 @@ this.Calendar.SelectionMode = CalendarSelectionMode.Multiple;
 {% endhighlight %}  
 {% endtabs %}
 
-![change-selection-mode](images/getting-started/maui-calendar-multiple-selection.png)
\ No newline at end of file
+![change-selection-mode](images/getting-started/maui-calendar-multiple-selection.png)
+
+## Action buttons
+
+You can display action buttons at the bottom of the calendar by using the `ShowActionButtons` property of SfCalendar. It allows you to confirm or cancel the selected date, dates, or date ranges of the `SfCalendar`.
+
+## Today button
+
+The today button can be displayed at the bottom of the calendar by using the `ShowTodayButton` property of the `SfCalendar`, allowing you to switch from the current view to today's view effortlessly.
```

Comment:
The today button can be displayed at the bottom of the calendar using the `ShowTodayButton` property of the `SfCalendar,` allowing you to switch from the current view to today's view effortlessly.

Reply:
Content modified.